### PR TITLE
Yarn is missing?

### PR DESCRIPTION
**Pull request by Spy-Seth**

### CODE REVIEW

**Comment by vvo** [comment]
I think that's just travis being weird from time to time

**Comment by vvo** [comment]
https://twitter.com/itsasbreuk/status/885646232298950656

File: .travis.yml [hunk 1]
```diff
@@ -5,3 +5,9 @@ branches:
     - master
     - /^greenkeeper\/.*$/
 cache: yarn
+
+# Since the upgrade to Ubuntu Precise, yarn is not available to run scripts: `yarn run foo`
+# So let's install it manually
+before_install:
```
Comment (vvo) [inline .travis.yml]: Any reference to this?

Comment (Spy-Seth) [inline .travis.yml]: https://docs.travis-ci.com/user/languages/javascript-with-nodejs/#Travis-CI-supports-yarn

**Comment by Spy-Seth** [comment]
> https://twitter.com/itsasbreuk/status/885646232298950656

I've try to empty the cache, and we already use `yarn`... so I guess it's a Travis issue 😭

File: .travis.yml (hunk 1 above)
Comment (vvo) [inline .travis.yml]: I am confused, this link says that by default Yarn is present when yarn.lock is here.

Comment (Spy-Seth) [inline .travis.yml]: Yes this is probably a Travis bug in the latest environment upgrade: https://blog.travis-ci.com/2017-07-11-trusty-as-default-linux-is-coming

**Pull request merged**